From d1a0ad61ae460d795d500a3245fddde41c7e3c0f Mon Sep 17 00:00:00 2001
From: Xavier Boniface <xavierx.boniface@intel.com>
Date: Thu, 2 May 2013 16:25:31 +0200
Subject: [PORT FROM MAIN][BT] Add bdt to userdebug builds

BZ: 104787

Needed by PPD because Bluetooth RF certification tests must be ran on
userdebug builds.

Note: you need to do an 'adb root' prior to run bdt on userdebug builds.
Otherwise, you will encounter some perimission issues.

Change-Id: I9045c3b22d7713df3e3aad5d616c9ca042359446
Orig-Change-Id: I1f8423565689f97d9816e10dcf68b189271cb0cc
Category: aosp improvement
Domain: CWS.BT-Common
Origin: internal
Upstream-Candidate: no, not-aosp
Signed-off-by: Xavier Boniface <xavierx.boniface@intel.com>
Reviewed-on: http://android.intel.com:8080/109275
Reviewed-by: Bondier, CedricX <cedricx.bondier@intel.com>
Tested-by: Zurmely, PierreX <pierrex.zurmely@intel.com>
---
 test/bluedroidtest/Android.mk |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/test/bluedroidtest/Android.mk b/test/bluedroidtest/Android.mk
index 0c6693a..9f79f26 100644
--- a/test/bluedroidtest/Android.mk
+++ b/test/bluedroidtest/Android.mk
@@ -23,7 +23,7 @@ LOCAL_SRC_FILES:=     \
 
 LOCAL_C_INCLUDES :=
 
-LOCAL_MODULE_TAGS := eng
+LOCAL_MODULE_TAGS := eng debug
 
 LOCAL_MODULE:= bdt
 
-- 
1.7.4.1

